tests/chisel.spec.ts

```diff
@@ -0,0 +1,129 @@
+import * as rimraf  from "rimraf";
+import * as PropertiesReader from "properties-reader";
+import { writeProperties } from "../src/utils";
+
+describe("commands unit tests", () => {
+    let chiselTestDataPath = "";
+    let tempFilePath: string;
+
+    /* important: 
+       VCAP_SERVICES contains special charecters and also \n that may cause to an issue in the a string 
+       when we read the text from the file the propertyReader add to the string "\" before each "\n" (e.g.: \\n)  
+       therefore we must define the VCAP service with \\n 
+    */   
+    const VCAP_SERVICE_WITH_ALL_CHARS = '{"hana":[{"name":"test-service","instance_name":"test-service","label":"hana","tags":["hana","database","relational","mta-resource-name:hdi_hdb","endpoint:https://api.cf.test.hana.ondemand.com","org:org","space:space"],"plan":"hdi-shared","credentials":{"certificate":"-----BEGIN CERTIFICATE-----\\nMIIDrzCCApegAwIBAgIQCDvgVpBCRrGhdWrJWZHHSjANBgkqhkiG9w0BAQUFADBh\\nMQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMRkwFwYDVQQLExB3\\nd3cuZGlnaWNlcnQuY29tMSAwHgYDVQQDExdEaWdpQ2VydCBHbG9iYWwgUm9vdCBD\\nQTAeFw0wNjExMTAwMDAwMDBaFw0zMTExMTAwMDAwMDBaMGExCzAJBgNVBAYTAlVT\\nMRUwEwYDVQQKEwxEaWdpQ2VydCBJbmMxGTAXBgNVBAsTEHd3dy5kaWdpY2VydC5j\\nb20xIDAeBgNVBAMTF0RpZ2lDZXJ0IEdsb2JhbCBSb290IENBMIIBIjANBgkqhkiG\\n9w0BAQEFAAOCAQ8AMIIBCgKCAQEA4jvhEXLeqKTTo1eqUKKPC3eQyaKl7hLOllsB\\nCSDMAZOnTjC3U/dDxGkAV53ijSLdhwZAAIEJzs4bg7/fzTtxRuLWZscFs3YnFo97\\nnh6Vfe63SKMI2tavegw5BmV/Sl0fvBf4q77uKNd0f3p4mVmFaG5cIzJLv07A6Fpt\\n43C/dxC//AH2hdmoRBBYMql1GNXRor5H4idq9Joz+EkIYIvUX7Q6hL+hqkpMfT7P\\nT19sdl6gSzeRntwi5m3OFBqOasv+zbMUZBfHWymeMr/y7vrTC0LUq7dBMtoM1O/4\\ngdW7jVg/tRvoSSiicNoxBN33shbyTApOB6jtSj1etX+jkMOvJwIDAQABo2MwYTAO\\nBgNVHQ8BAf8EBAMCAYYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUA95QNVbR\\nTLtm8KPiGxvDl7I90VUwHwYDVR0jBBgwFoAUA95QNVbRTLtm8KPiGxvDl7I90VUw\\nDQYJKoZIhvcNAQEFBQADggEBAMucN6pIExIK+t1EnE9SsPTfrgT1eXkIoyQY/Esr\\nhMAtudXH/vTBH1jLuG2cenTnmCmrEbXjcKChzUyImZOMkXDiqw8cvpOp/2PV5Adg\\n06O/nVsJ8dWO41P0jmP6P6fbtGbfYmbW0W5BjfIttep3Sp+dWOIrWcBAI+0tKIJF\\nPnlUkiaY4IBIqDfv8NZ5YBberOgOzW6sRBc4L0na4UU+Krk2U886UAb3LujEV0ls\\nYSEY1QSteDwsOoBrp+uvFRTp2InBuThs4pFsiv9kuXclVzDAGySj4dzp30d8tbQk\\nCAUw7C29C79Fv1C5qfPrmAESrciIxpg0X40KPMbp1ZWVbd4=\\n-----END CERTIFICATE-----\\n","driver":"com.sap.db.jdbc.Driver","hdi_password":"hdi_password_value","hdi_user":"hdi_user","host":"host_value","password":"password_value","port":"8080","schema":"schema_key","service_key_name":"SharedDevKey","url":"jdbc:sap://zeus.hana.canary.eu-central-1.whitney.dbaas.ondemand.com:36603?encrypt=true\u0026validateCertificate=true\u0026currentschema=2A33FD2D26B243188787413E69DEB742","user":"user_value"}}]}';
+    const envWithChisel = {   
+        "TUNNEL_PARAM": "2020:127.0.0.1:2020",
+        "CHISEL_URL": "https://chise-server.cfapps.test.hana.ondemand.com",
+        "CHISEL_USER": "chisel-user",
+        "CHISEL_PASSWORD": "chisel-password",
+        "VCAP_SERVICES": VCAP_SERVICE_WITH_ALL_CHARS
+    };
```

Comment:
it is a public repo - please remove or fake the whole actual data ( cretificates/url/id(s) )

Reply:
the data are fake,  
I keep it with \n because that was the issue when loading from the file